### ISSUE TITLE
Why seg dice loss never goes down?

### ISSUE THREAD

**Issue by andys0975** [MIC-DKFZ/medicaldetectiontoolkit#6]
The Retina UNet's seg dice loss never goes down while training
![image](https://user-images.githubusercontent.com/21152737/49907733-7972ca80-feb2-11e8-8974-56be411f9147.png)

And the validation segmentation predictions are always messed up like below row
![image](https://user-images.githubusercontent.com/21152737/49907869-fc942080-feb2-11e8-8cc5-468f52aee7de.png)

**Comment by pfjaeger** [comment]
Hi thanks for reaching out. What’s your input data? For me the Retina Unet started training pretty early without much tuning. Did you try without batch norm (I.e. no feature norm at all)? And for different learning rates?

**Comment by andys0975** [comment]
Thanks for your response.
I'm using BraTS18 Brain tumor data with good delineation of contour, and as you can see in my another issue, the ground truth row performs normal, however the model inference shows square like masks, and the seg dice loss never goes down even training all 20000 iterations. BTW, I use instance norm not batch norm

**Comment by andys0975** [comment]
I‘ll try using no norm and other parameter setting, but I still doubt there may be some problems when I use my own dataloader？

**Comment by pfjaeger** [comment]
did you solve this? please be more specific about problems in order to receive useful feedback

**Comment by andys0975** [comment]
No, still the same situation. I started to build my own architechture myself.
Maybe you can test your toolkit on other open datasets and share some experiences.

**Comment by pfjaeger** [comment]
The architecture is not the problem. We (or specifically our co-author Fabian Isensee) have succesfully participated in the BraTS challenge in recent years, training u-nets with dice loss. There is no conceptual difference for the dice loss function here, so no reason why your segmentation outputs should look like this. You would need to look at hyper parameter settings or preprocessing and input normalization schemes. I would like to help, but you would need to be more specific about your set up.

**Comment by amberhappy** [comment]
Hi,I have same problem on Lung dataset, the seg_dice_loss in FPN are always be near 1, and I tried to split FPN with the other 2 subnet, only train FPN net alone. But got seg_loss near 1 two. 
And change Norm from 'None' to 'instance_norm' does't help.
